doc/source/actors.rst

```diff
@@ -10,32 +10,63 @@ that specific worker and can access and mutate the state of that worker.
 Creating an actor
 -----------------
 
-You can convert a standard Python class into a Ray actor class as follows:
+.. tabs::
+  .. group-tab:: Python
 
-.. code-block:: python
+    You can convert a standard Python class into a Ray actor class as follows:
 
-  @ray.remote
-  class Counter(object):
-      def __init__(self):
-          self.value = 0
+    .. code-block:: python
 
-      def increment(self):
-          self.value += 1
-          return self.value
+      @ray.remote
+      class Counter(object):
+          def __init__(self):
+              self.value = 0
 
-Note that the above is equivalent to the following:
+          def increment(self):
+              self.value += 1
+              return self.value
 
-.. code-block:: python
+    Note that the above is equivalent to the following:
 
-  class Counter(object):
-      def __init__(self):
-          self.value = 0
+    .. code-block:: python
 
-      def increment(self):
-          self.value += 1
-          return self.value
+      class Counter(object):
+          def __init__(self):
+              self.value = 0
 
-  Counter = ray.remote(Counter)
+          def increment(self):
+              self.value += 1
+              return self.value
+
+      Counter = ray.remote(Counter)
```

Comment:
Not related with this PR, but we should also add the example code about how to create an actor here.